pepys_admin/admin_cli.py

```diff
@@ -12,8 +13,9 @@
 class AdminShell(cmd.Cmd):
     intro = """--- Menu ---
 (1) Export
-(2) Initialise/Clear
-(3) Status

```

Comment:
@BarisSari - I think we've actually implemented "Export by platform and sensor" (which is quite ok).  So, Option 2 should be changed, please.

Reply:
Changed:
https://github.com/debrief/pepys-import/pull/266/commits/e4fdc59f34cdfaa142c9dcb5bb582efeffb3e985